assets/xml/boot/fault_drawer.xml

```diff
@@ -0,0 +1,6 @@
+<Root>
+    <File Name="boot" OutName="fault_drawer_font" BaseAddress="0x80080060" RangeStart="0x18BF0" RangeEnd="0x18FF0" >
+        <!-- The real version is const and found in fault_drawer.c, though shares the same name -->
```

Comment:
Would like opinions on this. Is this acceptable, any ideas what to name the unused extracted variable instead?

Reply:
I think it is fine this way